contracts/epochs/FlowEpoch.cdc

```diff
@@ -466,6 +802,7 @@ access(all) contract FlowEpoch {
                     "Invalid startView, stakingEndView, and endView configuration"
             }
 
+            // TODO: replace with stopEpochComponents?
```

Comment:
We need the functionality, because we will still have sporks. We could refactor so that `reset` and `recover` use the same function (with some option to emit the EpochRecover event or not). 

My inclination is to keep it as-is though. We already know it works for the process it is used for.